packages/keymaps/src/browser/keymaps-frontend-contribution.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { inject, injectable } from 'inversify';
+import {
+    CommandContribution,
+    Command,
+    CommandRegistry,
+    MenuContribution,
+    MenuModelRegistry
+} from '@theia/core/lib/common';
+import { open, OpenerService } from '@theia/core/lib/browser';
+import { FrontendApplication } from '@theia/core/lib/browser';
+import { CommonMenus } from "@theia/core/lib/browser/common-frontend-contribution";
+import { WidgetManager } from '@theia/core/lib/browser/widget-manager';
+import { keymapsUri } from './keymaps-service';
+export namespace KeymapsCommands {
+    export const OPEN_KEYMAPS: Command = {
+        id: 'keymaps:open',
+        label: 'Open Keyboard Shortcuts'
```

Comment:
`Open Keyboard Shortcuts` would open an empty file. I think that is a bit surprising for users.
Should we maybe generate a json based on the defaults on first access and then use some diff tool to find the differences?
Also the we use auto-save by default, so there will be frequent contentChange signals on the resource with broken syntax.

Reply:
Yea we just discussed this, I think maybe for now opening two editors side by side would be nice. The one on the left with the default keybindings provided to the registry (readonly) and the one on the right the custom keymaps.json. A sort of diff editor would be nice, but I don't want it to show all the differences like a default diff editor would do... 

I'm not sure how to resolve the frequence contentChange signals problem, isn't that a common problem with using that setting anyway?

Reply:
Also can we do a separate PR for opening the default shortcuts?

Reply:
> Also can we do a separate PR for opening the default shortcuts?

Yes, let's do that